app/routes/admin/groups/components/GroupMembers.js

```diff
@@ -47,10 +63,23 @@ function mapStateToProps(state, props) {
     groupId: props.params.groupId
   });
 
-  return { memberships };
+  return {
+    memberships,
+    fetching: state.memberships.fetching,
+    hasMore: get(state, [
+      'memberships',
+      'pagination',
+      props.group && props.group.id,
```

Comment:
this `&&` might as well be outside the `get` right? `props.group && get()` or defined as `groupId` and `groupId && get()` if flow complains.